webui/react/src/components/kit/utils/components/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,21 @@
+.base {
```

Comment:
should the contents of `components/kit/utils/components` just be in `components/kit` ?

Reply:
open to rethinking this to avoid the annoying path names, but the intent was to clearly differentiate between "private" components which are dependencies that have been copied from elsewhere in the determined repo, and "public" components which are ready to be used as part of the actual UI kit.

many of the components in `utils/components` should be moved up to `kit` at some point anyway, but I figured that should be tackled one component at a time instead of all in this PR.

Reply:
ok makes sense. maybe `kit/internal/components`, `kit/internal/utils`, etc?

Reply:
Seconding `internal`, that's the standard naming for stuff like this.

Reply:
Seconding internal, that's the standard naming for stuff like this.